WMF Framework/Widget/WidgetContentFetcher.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+public final class WidgetContentFetcher {
+
+	// MARK: - Nested Types
+
+	public enum FetcherError: Error {
+		case urlFailure
+		case contentFailure
+		case unsupportedLanguage
+	}
+
+	public typealias FeaturedContentResult = Result<WidgetFeaturedContent, WidgetContentFetcher.FetcherError>
+
+	// MARK: - Properties
+
+	public static let shared = WidgetContentFetcher()
+
+	let session = URLSession.shared
```

Comment:
👍🏽 I've updated this fetcher to use `Session`. I initially used a bare `URLSession` just to avoid any unintended consequences, but it seems reasonable to use `Session` instead and get all the freebies (like the `Accept-Language` header).